apps/commudle-admin/src/app/app-shared-components/newsletter/newsletter.component.html

```diff
@@ -1,222 +1,224 @@
-<div class="main">
-  <div class="create-button">
-    <button nbButton status="primary" [routerLink]="['new']">
-      <fa-icon [icon]="icons.faPlus" class="com-mr-2"></fa-icon>Create Newsletter
-    </button>
-  </div>
+<div class="newsletter-wrapper">
```

Comment:
ng template should not inside any div and the method to use this class was not correct